applications/DEM_application/custom_elements/rigid_body_element.h

```diff
@@ -88,32 +86,39 @@ namespace Kratos {
         {
 	    //mpGeometry->PrintData(rOStream);
         }
+        
+        std::vector<array_1d<double, 3> > mListOfCoordinates;
+        std::vector<Node<3>::Pointer > mListOfNodes;
```

Comment:
Do we need a ListOfCoordinates and a ListOfNodes?

Reply:
We need both, the list of initial coordinates is necessary to update the Orientation

Reply:
But the nodes contain the initial coordinates. X0, Y0, Z0 or GetInitialPosition should do the job. Would it work?

Reply:
Ok! I will try to change it in that way.

Reply:
mListOfCoordinates is necessary because it stores the local coordinates of the nodes that compose the Rigid Element.